src/pt/BreadCrumbs.jl

```diff
@@ -0,0 +1,62 @@
+"""
+A struct that provides a basic, user-friendly interface to Pigeons. Only two inputs
+are required, in positional order:
+$FIELDS
+
+!!! note
+
+    The PT state is initialized using a random sample from the reference.  
+"""
+struct BreadCrumbs{TRefDist <: Distributions.Distribution, TTarget}
```

Comment:
Can you use @auto instead? Otherwise the standard out gets cluttered with pages of mostly useless type information when showing the stack trace. 

Also ref should be more general. We don't want Distributions to only work in BreadCrumbs. It should be seamless with the `reference = ..` method as well. (unless we remove the reference = .. method; which I don't think is a good idea since it works well for PPLs). 

Conversely, things that can be fed into `reference = ..` should work in the target_log_potential argument. Similarly things that gets fed into `target = ` should work in target_log_potential (I think this is already the case?).